doc/ipythonconsole.rst

```diff
@@ -2,91 +2,145 @@
 IPython Console
 ###############
 
-The **IPython Console** allows you to execute commands and enter, interact with and visualize data inside any number of fully featured `IPython`_ interpreters.
-Each console is executed in a separate process, allowing you to run scripts, interrupt execution and restart or terminate a shell without affecting the others or Spyder itself, and easily test your code in a clean environment without disrupting your primary session.
+The **IPython Console** allows you to execute commands and enter, interact with and visualize data inside `IPython`_ interpreters.
 
 .. _IPython: https://ipython.org/
 
 .. image:: images/console/console-standard.png
    :alt: Spyder IPython Console with code, inline plots, and the In prompt
 
+To launch a new IPython instance, go to :guilabel:`New console (default settings)` under the :guilabel:`Consoles` menu, the :guilabel:`IPython Console` pane menu or its context menu, or use the keyboard shortcut (:kbd:`Ctrl-T` by default).
 
-=======================
-Connecting to a console
-=======================
+.. image:: images/console/console-new.gif
+   :alt: Spyder showing opening a new IPython Console
+
+From any of the three menus, you can stop currently executing code with :guilabel:`Interrupt kernel`, clear a console's namespace with :guilabel:`Remove all variables`, or relaunch a fresh one with :guilabel:`Restart`.
+As each Console is executed in a separate process, this won't affect the others or Spyder itself, and you will be able to easily test your code in a clean environment without disrupting your primary session.
```

Comment:
> I don't think it's necessary to capitalize console.

Yeah, I decided on the convention of Console to refer to the pane and console to refer to the individual consoles a while back, but was really sloppy about following it here. Good catch.

> I don't see the need to mention that opening new consoles can affect Spyder itself.

I guess the intention was that it could lock up the program (as used to happen in Rstudio), but if this isn't a common user expectation these days then we can leave it out.

> Instead, I'd prefer to mention that new consoles won't affect the previous ones.

Yes, but anything that happens in that console also won't affect ones opened after it, so I'm not sure why it makes sense to single out only ones opened before; this sentence isn't just about opening consoles but working with them in general, and (per the previous sentence) running, stopping, restarting, etc in particular. Therefore, I've revised your proposal to reflect this, as well as a 1 <= n < inf number of consoles (instead of 2 <= n < inf).

```suggestion
As each console is executed in a separate process, this won't affect any others you've opened, and you will be able to easily test your code in a clean environment without disrupting your primary session.
```

Reply:
> Sidenote: I haven't thought about this point before, but now I see that it's very important to mention it. Thanks @CAM-Gerlach for adding it to the docs!

Its been two years since we added this so I don't remember the exact circumstances, but there's a good chance it was _you_ that suggested to _me_ that I mention this, heh.